.github/workflows/golangci-lint.yaml

```diff
@@ -0,0 +1,26 @@
+name: CI Linter with golangci-lint
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    runs-on: ubuntu-20.04
```

Comment:
nit: let's try to run with `ubuntu-22.04` which is the most updated host version provided by GH actions nowdays

Reply:
thanks for the comment. 
changed.